jbpm-human-task/jbpm-human-task-workitems/pom.xml

```diff
@@ -95,6 +100,11 @@
 	  <artifactId>jbpm-human-task-jpa</artifactId>
 	  <scope>test</scope>
 	</dependency>
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-persistence-jpa</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
some are added for the transient enforcer check - I will remove it and run a build and see if it's still required